jenkins/run-pytests

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+prog="$(basename "${0}")"
+progdir="$(realpath -e $(dirname "${0}")/..)"
+
+source $(dirname "${0}")/python-setup.sh
+
+printf -- "black --check ...\n"
+black --check .
+black_sts=${?}
+if [[ ${black_sts} -ne 0 ]]; then
+    printf -- "FAILED: black --check .\n" >&2
+fi
+printf -- "\nflake8 --verbose ...\n"
+flake8 --verbose .
+flake_sts=${?}
+if [[ ${flake_sts} -ne 0 ]]; then
+    printf -- "FAILED: flake8 --verbose .\n" >&2
+fi
+printf -- "\n"
+pytest {posargs} ./server/lib/pbench/test/unit
+pytest_sts=${?}
+if [[ ${pytest_sts} -ne 0 ]]; then
+    printf -- "FAILED: pytest ...\n" >&2
+fi
+final_sts=$((black_sts + flake_sts + pytest_sts))
+exit ${final_sts}
```

Comment:
This appears to be a clean copy of what's on `main` with the exception of the 0.69 pytest path, so I won't offer several possible simplifications (for example, using a `flake="flake8 --verbose ."` env variable instead of repeating the string three times for the header, command execution, and failure message).

However, I will point out that the file ought to end with a newline. 😁